README.md

```diff
@@ -35,6 +35,54 @@ Contribute your own observability pack to the New Relic One catalog by following
 
     This process is similar for all other entity directories. Also, if you don't want to create entities for a given type, delete the corresponding directory.
 
+    ## Using NerdGraph for existing alerts
+
+    When adding alerts to your quickstart, if you want to start from an existing alert, you can use the [Graphql API](https://api.newrelic.com/graphiql) to get json object that can assit you populating your `alerts.yaml` file.
+
+    For example, to add an existing alert policy with a `NrqlBaselineCondition`, the following NerdGraph query will assist in `alerts.yaml` file:
```

Comment:
the following NerdGraph query will assist in `alerts.yaml` file -> this is a bit of a weird sentence
suggestion: the following NerdGraph query will return a json object that you can use to populate the `alerts.yaml` file